### DISCUSSION TITLE
Export HTML from JS to separate file(s)

### DISCUSSION THREAD

**Discussion by hmalaud** [statamic-rad-pack/shopify#72] [Feature Requests]
In addition of being easier to read, this would allow updates to JS files without erasing user modifications in the HTML strings.

**Comment by jackabox** [comment]
Not really sure what you mean on this one.

**Reply by hmalaud** [reply]
Right now, HTML for banner and product rows in the cart are stored in strings in the middle of HTML files (cart.js and products.js), it would be easier to have them in separate files (e.g. "product-row.js" and "banner.js") to separate them from the logic and have the possibility to update the whole addon without erasing custom HTML the user might have made (chances are an addon update will more likely change the logic but not the templates). 
I hope I made it clearer this time, don't hesitate to tell me if not :)

**Reply by jackabox** [reply]
Gotcha, makes sense.

With Statamic 3.2 we should have a way to easily import/export starter kits, and I'm hoping to move this to a more reusable way within the starter kit. If that doesn't work how I'd expect, then this would be a great change :)
